### DISCUSSION TITLE
How to remove a risk from the risk analysis?

### DISCUSSION THREAD

**Discussion by danje57** [monarc-project/MonarcAppFO#422] [Q&A] [answered]
Dear all,

Sorry in advance, if I dn't see the option on the interface.

However, I would like to remove 2 identified risks from my analysis.

But I don't see the option...

![image](https://user-images.githubusercontent.com/16591723/160575285-d75e1992-b130-47f7-83e3-d599867ea8fb.png)

Do you have a tips for that?

Regards

**Comment by jerolomb** [comment, marked as answer]
Hello,

in the treeview of your risk analysis click on building, and normally the icon to delete will appear. Same for Building PCI-DSS.

Tell us if it's not the case. 

Regards

**Reply by danje57** [reply]
Ohhh yes I'm stupid :p that's fine!!! Thanks!
